esrally/client/common.py

```diff
@@ -0,0 +1,58 @@
+import re
+from datetime import date, datetime
+from typing import Any, Mapping
+
+from elastic_transport.client_utils import percent_encode
+from elasticsearch import VERSION
+
+
+def _client_major_version_to_str(version: tuple) -> str:
+    return ".".join(map(str, version)).partition(".")[0]
```

Comment:
This can be simplified, I'm not sure a separate function is needed:

```suggestion
def _client_major_version_to_str(version: tuple) -> str:
    return str(version[0])
```

Reply:
Addressed in https://github.com/elastic/rally/pull/1669/commits/d1a911fec02dd4b6d48dce8ccb4bb076630f0e2f